src/services/prismic.js

```diff
@@ -45,3 +50,23 @@ export type Document<T> = {
 export function queryDocumentsByType<T>(type: string, options?: QueryOptions): Response<T> {
   return prismicClient.query(Prismic.Predicates.at(DOCUMENT_TYPE, type), options);
 }
+
+export async function queryDocumentsByID(id: string): Promise<Object> {
```

Comment:
It looks like the result is string?
```suggestion
export async function queryDocumentsByID(id: string): Promise<string> {
```